src/js_realm.hpp

```diff
@@ -206,6 +209,7 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
         {"schemaVersion", wrap<schema_version>},
         {"clearTestState", wrap<clear_test_state>},
         {"copyBundledRealmFiles", wrap<copy_bundled_realm_files>},
+        {"_waitForDownload", wrap<wait_for_download_completion>},
```

Comment:
Check this comment on the underscore name. https://github.com/realm/realm-js/pull/1005#issuecomment-300915003